### PR TITLE
Add confirmation dialog when deleting target

### PR DESCRIPTION
**Pull request by marcossalto**
Change background color on select target
Show modal dialog on delete target
Refresh map

Screenshot_1615323877
Screenshot_1615323883
Screenshot_1615323890

### CODE REVIEW

File: app/src/main/java/com/marcossalto/targetmvd/ui/target/MapFragment.kt [hunk 1]
```diff
@@ -237,6 +257,27 @@ class MapFragment : PermissionFragment(), OnMapReadyCallback {
         }
     }
 
+    private fun selectTargetMarker(target: TargetModel) {
+        lifecycleScope.launch {
+            selectedTarget?.topic?.run{
+                targetModelMap[selectedTarget]?.setIcon(getMarkerDrawable(selectedTarget!!,false))
```
Comment (agustinkoll-rootstrap) [inline app/src/main/java/com/marcossalto/targetmvd/ui/target/MapFragment.kt]: avoid using not null asertion (!!)

Comment (agustinkoll-rootstrap) [reply app/src/main/java/com/marcossalto/targetmvd/ui/target/MapFragment.kt]: you can use it

File: app/src/main/java/com/marcossalto/targetmvd/ui/target/MapFragment.kt [hunk 2]
```diff
@@ -237,6 +257,27 @@ class MapFragment : PermissionFragment(), OnMapReadyCallback {
...
+            }
+            target.topic.run {
+                selectedTarget = target
+                targetModelMap[target]?.setIcon(getMarkerDrawable(target,true))
+            }
+        }
+    }
+
+    private suspend fun getMarkerDrawable(target: TargetModel, b: Boolean) : BitmapDescriptor? {
```
Comment (agustinkoll-rootstrap) [inline app/src/main/java/com/marcossalto/targetmvd/ui/target/MapFragment.kt]: here you can use a more significant name for b

**Review by agustinkoll-rootstrap — APPROVED** [review]

File: app/src/main/java/com/marcossalto/targetmvd/ui/target/MapFragment.kt (hunk 1 above)
Comment (marcossalto) [reply app/src/main/java/com/marcossalto/targetmvd/ui/target/MapFragment.kt]: Thanks for your suggestion

File: app/src/main/java/com/marcossalto/targetmvd/ui/target/MapFragment.kt (hunk 2 above)
Comment (marcossalto) [reply app/src/main/java/com/marcossalto/targetmvd/ui/target/MapFragment.kt]: Thanks for your suggestion

File: app/src/main/res/layout/layout_delete_confirmation.xml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    <TextView
+        android:id="@+id/warning_message_confirm_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/no_dimen"
+        android:layout_marginTop="@dimen/no_dimen"
+        android:layout_marginEnd="@dimen/no_dimen"
+        android:padding="@dimen/padding_30"
+        android:text="Remember that if you delete it, you won’t be able to chat with it’s matches anymore"
```
Comment (ximenaperez) [inline app/src/main/res/layout/layout_delete_confirmation.xml]: Remember to add the texts to the strings file

Comment (marcossalto) [reply app/src/main/res/layout/layout_delete_confirmation.xml]: Thanks for your suggestion.

**Review by amaury901130 — APPROVED** [review]

File: app/src/main/res/layout/layout_delete_confirmation.xml (hunk 3 above)
Comment (marcossalto) [reply app/src/main/res/layout/layout_delete_confirmation.xml]: Thanks for your suggestion

**Pull request merged**